src/signals/incident-management/containers/IncidentDetail/components/StatusForm/styled.tsx

```diff
@@ -2,85 +2,114 @@
 // Copyright (C) 2020 - 2021 Gemeente Amsterdam
 import styled from 'styled-components'
 import {
+  Alert,
   Heading,
-  themeSpacing,
+  Label,
   Row,
-  Column,
   themeColor,
-  Label,
+  themeSpacing,
 } from '@amsterdam/asc-ui'
 
 import Button from 'components/Button'
 
+export const AddNoteWrapper = styled.div`
+  label {
+    display: none;
+  }
+  section div textarea {
+    padding: ${themeSpacing(3)};
+    margin-top: 0;
+    border-top: 1px solid transparent;
+    :hover {
+      border-top-color: transparent;
+    }
+  }
+`
+
 export const Form = styled.form`
   position: relative;
+  padding: ${themeSpacing(0, 5, 6, 5)};
+  margin-bottom: ${themeSpacing(6)};
+  background-color: ${themeColor('tint', 'level2')};
   width: 100%;
   grid-template-areas:
     'header'
     'options'
     'texts'
     'form';
-  grid-column-gap: 20px;
   display: grid;
-  margin-bottom: 5em;
   grid-row-gap: ${themeSpacing(4)};
+`
 
-  @media (min-width: ${({ theme }) => theme.layouts.medium.max}px) {
-    grid-template-columns: 6fr 6fr;
-    grid-template-areas:
-      'header texts'
-      'options texts'
-      'form texts'
-      '. texts';
-  }
+export const FormArea = styled.div`
+  grid-area: form;
+  display: grid;
```

Comment:
I don't it's necessary to re-declare grid properties on a nested grid element.

Reply:
formArea is now removed

Reply:
And the other grid stuff in the form